### PR TITLE
update colors and whats new page

### PR DESCRIPTION
**Pull request by britt6612**
#### What does this PR do?
This PR  adds to the what's new page and updates the color page to include validation-ok and warning
#### Where should the reviewer start?
colors.js & whatsnew.mdx
#### What testing has been done on this PR?
none
#### How should this be manually tested?
none
#### Any background context you want to provide?
none
#### What are the relevant issues?
Colors for `validation-warning` and `validation-ok` were finalized from a design point in https://github.com/grommet/hpe-design-system/issues/1105

- [x] Add `validation-warning` and `validation-ok` to HPE theme
- [x] Add and publish to colors page
- [x] Add entry to "What's New Page"
#### Should this PR be mentioned in Design System updates?
no
#### Is this change backwards compatible or is it a breaking change?
backward compatible

### CODE REVIEW

File: aries-site/src/data/color.js [hunk 1]
```diff
@@ -243,6 +243,16 @@ const inputLight = [
     value: colors['validation-critical'].light,
     hex: colors['validation-critical'].light,
   },
+  {
+    name: 'validation-ok',
+    value: colors['validation-critical'].light,
```
Comment (taysea) [inline aries-site/src/data/color.js:248]:
```suggestion
    value: colors['validation-ok'].light,
```
Change for all others too, same for `validation-warning`

**Review by taysea — CHANGES_REQUESTED** [review]
Need to change the value to the appropriate color name, right now they're all coming as `validation-critical`:
Screen Shot 2021-02-24 at 10 27 51 AM

File: aries-site/src/pages/whats-new.mdx [hunk 2]
```diff
@@ -53,6 +53,55 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
...
+- Aligned [layer component](https://github.com/grommet/hpe-design-system/pull/1434) to match the figma files. 
+
+## Febuary 8-12, 2021
+
+### Homepage 
+
+- Completed a fresh new look for our new HPE design system [home page](https://github.com/grommet/hpe-design-system/pull/1408).
+- [Figma](https://www.figma.com/file/CWB0xMmfVx6cT0ugzluySl/home-page?node-id=87%3A379) is updated as well 
+
+### Added Wizard themplate
```
Comment (taysea) [inline aries-site/src/pages/whats-new.mdx:85]:
```suggestion
### Updated Wizard template 
```

File: aries-site/src/pages/whats-new.mdx [hunk 3]
```diff
@@ -53,6 +53,55 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
...
+
+### Layer 
+
+- Aligned [layer component](https://github.com/grommet/hpe-design-system/pull/1434) to match the figma files. 
+
+## Febuary 8-12, 2021
+
+### Homepage 
+
+- Completed a fresh new look for our new HPE design system [home page](https://github.com/grommet/hpe-design-system/pull/1408).
```
Comment (taysea) [inline aries-site/src/pages/whats-new.mdx:82]:
```suggestion
- Completed a fresh new look for our new HPE Design System [home page](https://github.com/grommet/hpe-design-system/pull/1408).
```

File: aries-site/src/pages/whats-new.mdx [hunk 4]
```diff
@@ -53,6 +53,55 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
...
+
+### New FileInput Component
+- Check out the new [FileInput](https://v2.grommet.io/fileinput) Component on the grommet stable branch.
+
+### New Pagination Component 
+- The new [Pagination Component](https://v2.grommet.io/pagination) is ready to be used!
+
+### Re-align button sizes
+
+- Adjusted Button sizes to use [primary](https://github.com/grommet/hpe-design-system/pull/1433).
```
Comment (taysea) [inline aries-site/src/pages/whats-new.mdx:72]: I don't think we need to share this one, it's just regarding a tiny site example but not really a change someone would see in their app

File: aries-site/src/pages/whats-new.mdx [hunk 5]
```diff
@@ -53,6 +53,55 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
...
+
+The new Spinner component has been released on grommet stable [branch](https://github.com/grommet/grommet/pull/4981).
+
+## Febuary 15-19, 2021
+
+### New FileInput Component
+- Check out the new [FileInput](https://v2.grommet.io/fileinput) Component on the grommet stable branch.
+
+### New Pagination Component 
+- The new [Pagination Component](https://v2.grommet.io/pagination) is ready to be used!
```
Comment (taysea) [inline aries-site/src/pages/whats-new.mdx:68]:
```suggestion
- Check out the new [Pagination Component](https://v2.grommet.io/pagination) on the grommet stable branch.
```

File: aries-site/src/pages/whats-new.mdx (hunk 4 above)
Comment (britt6612) [reply aries-site/src/pages/whats-new.mdx:72]: okay that's fine I wasn't sure where we draw the line

File: aries-site/src/data/color.js (hunk 1 above)
Comment (britt6612) [reply aries-site/src/data/color.js:248]: yes, I got an error that's why I left them as critical looks like the theme has to be merged first to call the colors.

Sorry, should have made a comment about that.

File: aries-site/src/pages/whats-new.mdx (hunk 4 above)
Comment (taysea) [reply aries-site/src/pages/whats-new.mdx:72]: No worries! I think in general we are thinking about What's New as a sort of "weekly release notes" in a sense, so any change that will affect how a team's app might look/new guidance on how things should behave

**Review by taysea — APPROVED** [review]
LGTM.

**Pull request merged**